src/flows/components/header/Submit.tsx

```diff
@@ -25,9 +25,15 @@ export const Submit: FC = () => {
   const time = timeContext[id]
   const tr = !!time && time.range
 
+  const hasQueries = flow.data.all
```

Comment:
what does this do?

Reply:
I was gonna ask the same

Reply:
So glad you asked. We are using it to disable the `Submit` button down below (where it was moved from), but in this case it's being used to prevent the `submit` function from being called when it's mounted:

https://github.com/influxdata/ui/pull/157/files#diff-d80db76fbd0677f114efeb2cd097c4aeR32-R36

Basically, creating an empty metric selector pipe and refreshing the page was causing an empty query to run (which is an added charge), and was also returning an error because the query was empty